src/itemEditableInfo/ItemEditableInfo.story.mdx

```diff
@@ -1,29 +1,47 @@
 import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks'
 
 import { ItemEditableInfo } from './index'
+import { FocusVisibleProvider } from '../_utils/focusVisibleProvider'
 
 <Meta title="Items|ItemEditableInfo" />
 
-# ItemEditableInfo
+# ItemEditableInfo link
```

Comment:
I think you want to have a main title "ItemEditableInfo" and "Link" as a subtitle

```
# ItemEditableInfo

### link
```